doc/manual/config-basic.rst

```diff
@@ -119,7 +119,7 @@ statements that require non-standard evaluation.
 Checking your configuration
 ---------------------------
 From the front page the *Config checker* is available. This tool will
-do a basic sanity check of your DOMjudge setup and gives helpful hints
+do a basic coherence check of your DOMjudge setup and gives helpful hints
```

Comment:
Just use 'basic check'?

Reply:
I guess that or `basic validation check`.